codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/CustomizableOperationGenerator.kt

```diff
@@ -262,9 +276,32 @@ class CustomizableOperationGenerator(
                         .send_orchestrator_with_plugin(final_plugin)
                         .await
                 }
+
+                ##[doc(hidden)]
+                // This is a temporary method for testing. NEVER use it in production
+                pub fn request_time_for_tests(mut self, request_time: ::std::time::SystemTime) -> Self {
+                    let interceptor = #{TestParamsSetterInterceptor}::new(move |cfg: &mut #{ConfigBag}| {
+                        cfg.put(request_time);
+                    });
+                    self.interceptors.push(#{SharedInterceptor}::new(interceptor));
+                    self
+                }
+
+                ##[doc(hidden)]
+                // This is a temporary method for testing. NEVER use it in production
+                pub fn user_agent_for_tests(mut self) -> Self {
```

Comment:
This needs to get added via `sdk-codegen` since the `aws-http` crate isn't available for generic clients.

Reply:
CI failed as expected. Fixed in 54c83b9.